lib/views/pages/profile/profile_page.dart

```diff
@@ -42,8 +42,9 @@ class ProfilePage extends ConsumerWidget {
         ProfileInfoCard(
           content:
               "Here is a FAKE data comment on a super post that talks about something that is super cool and is located in a super spot that is very cosy and nice",
-          onDelete: () async {
```

Comment:
This change should be reverted. It was probably missed in 4905711.

Reply:
If I revert this line I will get a warning of type "redundant 'async' keyword" by DCM. Should we really add the async keyword here even if we don't await anything for now?

Reply:
Do not only revert this line, revert the whole changes for this file.

Do you still get a warning ?

Reply:
I reverted the file and and still get the warning. Should we leave the code like that?

Reply:
Yes, we will change this soon anyway as comment deletion is about to get implemented in UI.